Localization, styling and code cleanup in options page

whiver opened the pull request:
- Add localization to the only option available in the option page
- Fix the design to be consistent with the other pages
- Update the saving logic in the option page to return to the main page after save
- Remove the custom logic in the computed fields in PasswordGenerator: now I'm reusing the `resetPassword` action used when clicking on the "LessPass" menu link, which resets the password to defaultPassword.
- update `isDefaultProfile` to compare the current password with the defaultPassword from the store instead of the harcoded one, now that we can customize the defaults. It fixes the password options pane being open after modifying the options

whiver commented:
Hi :)
Question about translations: is there a specific process to add localized strings or should I just commit them? (I can add at least French translations for the options page).

guillaumevincent commented:
I removed netlify, so let me try to remove this from tests

guillaumevincent commented on packages/lesspass-pure/src/views/OptionsPage.vue:
```diff
@@ -1,20 +1,28 @@
         />
+      </div>
+    </div>
+    <div class="form-group row">
```
you don't need this row
just let the button alone without any div

guillaumevincent commented on packages/lesspass-pure/src/store/getters.js:
```diff
@@ -12,11 +12,10 @@ export const passwordURL = state => {
 
 export const isDefaultProfile = state => {
-  let defaultProfile = true;
   for (let key in defaultOptions) {
```
I think you want `let key in state.defaultPassword` no?

guillaumevincent commented on packages/lesspass-pure/src/store/getters.js:
```diff
@@ -12,11 +12,10 @@ export const passwordURL = state => {
     }
   }
-  return defaultProfile;
+  return true;
```
I think you want `let key in state.defaultPassword` no?

whiver replied:
That's what I did in the first time, but it's not consistent with the original behaviour, as the state.defaultPassword object also includes `login` and `site`.
Concretely, it would make the password options pane appear as soon as we update the default login in the options (`PasswordGenerator`, l. 104), which I guess is not the desired behaviour.

guillaumevincent commented:
some comments directly in the files

> Remove the custom logic in the computed fields in PasswordGenerator: now I'm reusing the resetPassword action used when clicking on the "LessPass" menu link, which resets the password to defaultPassword.

I noticed this today also, nice fix

> Question about translations: is there a specific process to add localized strings or should I just commit them? (I can add at least French translations for the options page).

just run `yarn build:i18n` and edit the new lines in `packages/lesspass-pure/src/i18n/*` files

whiver commented:
> some comments directly in the files
> 
> > Remove the custom logic in the computed fields in PasswordGenerator: now I'm reusing the resetPassword action used when clicking on the "LessPass" menu link, which resets the password to defaultPassword.
> 
> I noticed this today also, nice fix
> 
> > Question about translations: is there a specific process to add localized strings or should I just commit them? (I can add at least French translations for the options page).
> 
> just run `yarn build:i18n` and edit the new lines in `packages/lesspass-pure/src/i18n/*` files

Thanks, I updated the translations and addressed your comments :slightly_smiling_face:
